extensions/amp-ad-network-doubleclick-impl/0.1/test/test-doubleclick-a4a-config.js

```diff
@@ -56,33 +58,32 @@ describe('doubleclick-a4a-config', () => {
   });
 
   describe('#doubleclickIsA4AEnabled', () => {
```

Comment:
I am generally displeased that this class does not actually test any of the random selection code.  The tests just force a selection and verify it returns the correct value.  Given you're restructuring the class to ease testing, how would you feel about addressing this deficiency?